### PR TITLE
resolve version from $env on environment deployment

### PR DESCRIPTION
**Pull request by EyalPerry**

### CODE REVIEW

File: sign-version/index.js [hunk 1]
```diff
@@ -153,6 +169,7 @@ async function run() {
       datetime,
       author,
     });
+    await updateCiVersion(versionName);
```
Comment (nivsto) [inline sign-version/index.js:172]: The only thing I don't like about it is the fact that every time a developer needs to update his code he'll create a new version, and we'll store this version forever.
Let's think of a way to simplify it --> for example running sign-version nightly with the same version name.

Comment (EyalPerry) [reply sign-version/index.js:172]: or when I detect $ci I just resolve CI like I would in sign version, without actually creating it!

Comment (EyalPerry) [reply sign-version/index.js:172]: inf my free time I am working on making a base docker image with all our action code to unify it, which will make reuse of devops functions a whole lot easier..

Comment (nivsto) [reply sign-version/index.js:172]: Event better! Let's do that :-)

**Pull request merged**